Gemfile

```diff
@@ -6,4 +6,12 @@ gemspec
 
 group :test do
   gem "html-proofer", "~> 5.0"
+  gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw]
+  gem 'wdm', '>= 0.1.0'
+  gem 'jekyll-compose', group: [:jekyll_plugins]
+  gem 'base64'
+  gem 'jekyll', '~> 4.3.3'
+  gem 'csv'
 end
+
+gem "webrick", "~> 1.8"
```

Comment:
Why were they added?

Reply:
I suppose those are all optional, but basically warnings came up when using `bundle exec jekyll serve`

Reply:
Dependency updates are Jekyll's job, not the theme's.